lib/moribus/extensions/has_current_extension.rb

```diff
@@ -6,15 +6,27 @@ module HasCurrentExtension
       # Sets 'is_current' flag of overridden record to false, instead
       # of deleting it or setting foreign key to nil.
       def remove_target!(*)
+        # Use custom update to avoid running ActiveRecord optimistic locking
+        # and to avoid updating lock_version column:
+        klass = target.class
+
         if target.new_record?
           target.is_current = false
+          target.updated_at = Time.zone.now if klass.column_names.include?("updated_at")
```

Comment:
It's better to move `klass.column_names.include?("updated_at")` check to separate method.